### DISCUSSION TITLE
How Do I Get The CLI Installed?

### DISCUSSION THREAD

**Discussion by DeathStalker77** [JPersson77/LGTVCompanion#241] [Q&A]
I have a 2023 LG C2 (77") and I'd like to upgrade to WebOS23, but I don't want to lose the access to the Service Menu.  I understand that the functionality can still be gained with v23 using the CLI.  I saw posts indicating the Developer Mode was available in the Content Store, but v22 does not have it listed in the APPS (no "Content Store" available).

Thanks!

--- DS

**Comment by JPersson77** [comment]
Everything you need to access the service menu (i.e the command line interface) is installed along with the app, in the installation directory: c:/program files/lgtv companion/. 

You can send command line instructions to either of lgtv companion.exe or lgtvcli.exe. Lgtvcli.exe is a console application. No need to enter developer mode on the TV etc.

1) press WIN+R, type cmd into the dialog and then press enter to start the windows command prompt.
2) type into the command prompt: cd "c:/program files/lgtv companion/"

to revert to the old style service menu (non-truncated) on webOS23:
3) type into the command prompt: LGTVcli.exe -settings_other "{\"svcMenuFlag\": false}"

to show the service menu:
4) type into the command prompt: LGTVcli.exe -button IN_START
5) use the code 0413 when prompted on the TV

**Reply by DeathStalker77** [reply]
Oh, didn't know I had to have it hooked to a computer.......

**Reply by JPersson77** [reply]
Yep that is the way
